templates/item/partials/ability.hbs

```diff
@@ -9,7 +9,7 @@
   {{!-- TODO: Forced Movement --}}
 </fieldset>
 {{/inline}}
-{{formGroup systemFields.keywords value=system.keywords options=keywords}}
```

Comment:
Can also remove this part of preparePartContext in AbilityData

Reply:
Pushed. I assume that's what you meant.